src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -26,27 +28,27 @@
  *
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
-class CombinedStore implements StoreInterface, LoggerAwareInterface
+class CombinedStore implements PersistStoreInterface, BlockingStoreInterface, LoggerAwareInterface, StoreInterface
```

Comment:
If it does not supports `WaitAndSave` why do we implements `BlockingStoreInterface`?